### PR TITLE
add Project Harbor and Carvel and sort the list

### PR DESCRIPTION
**Pull request by OrlinVasilev**
Signed-off-by: OrlinVasilev <ovasilev@vmware.com>

#### ℹ️ Repository information

**The repository has**:

- [X] At least three issues with the good first issue label.
- [X] Detailed setup instructions for the project.
- [X] CONTRIBUTING.md
- [X] Actively maintained.

### CODE REVIEW

File: data/repositories.toml [hunk 1]
```diff
@@ -261,6 +261,7 @@ repositories = [
   'github.com/gnuradio/gnuradio',
   'github.com/go-gitea/gitea',
   'github.com/go-swagger/go-swagger',
+  'github.com/goharbor/harbor'
```
Comment (sanket-deepsource) [inline data/repositories.toml:264]:
```suggestion
  'github.com/goharbor/harbor',
```

**Review by sanket-deepsource — CHANGES_REQUESTED** [review]

**Comment by OrlinVasilev** [comment]
@sanket-deepsource  thank you fixed!

**Review by siddhant-deepsource — COMMENTED** [review]
@OrlinVasilev Thanks for adding these amazing projects. Had one issue though with [vmware-tanzu/carvel](https://github.com/vmware-tanzu/carvel) that it doesn't contain any issue with the label `good-first-issue`. It would be great if you can at least open 3 beginner friendly issues on this project for new contributors. The other changes look good. 💯

**Review by sanket-deepsource — APPROVED** [review]

**Review by sanket-deepsource — CHANGES_REQUESTED** [review]
Hey @OrlinVasilev! Thanks for your contribution. Looks like there are some conflicts. Do you mind resolving them?

**Review by sanket-deepsource — APPROVED** [review]

**Comment by OrlinVasilev** [comment]
Anyone to merge ? :)

**Comment by OrlinVasilev** [comment]
Thank you!

**Pull request merged**